### ISSUE TITLE
Questions about datasets

### ISSUE THREAD

**Issue by yoheyokubo** [Tongqing-Wei/DeepPBI-KG#1]
Thanks for such an interasting work. I have some questions about the datasets to reproduce the results and extend this work.

1. **Training and test set** (3647 interactions between 3513 phages and 395 bacteria):
Do you provide the results of the training and test split? In addition, regarding the following sentence in the paper "For bacteria that interact with >10 phages, the following procedure was employed: K-means was used to select the 10 most representative interacting phages for inclusion in the training set, with the remaining samples allocated to the test set", does that mean K-means was applied to _key gene feature vectors_ of those phages?

2. **Independent and external test set** (1230 interactions between 1182 phages and 166 bacteria):
Do you provide the interaction information about the phages and hosts in the repository [DeepPBI-KG_external_data](https://github.com/Tongqing-Wei/DeepPBI-KG_external_data), like _model/interaction_ref_infor.csv_? Furthermore, it seems that the directory _phage_external_ contains 1256 phages, while the directory _host_external_ contains 166 hosts correctly.

3. **K. pneumoniae (KP) dataset** (3169 interactions between 110 KP phages and 113 KP subspecies):
Do you provide the sequenced data, the interaction information, and the data-splitting information of them? After searching submission and accession numbers mentioned in the _Data availabity_ section in the paper, I found the [entries](https://ngdc.cncb.ac.cn/gwh/search/advanced/result?search_category=&search_term=&source=0&query_box=PRJCA025410) corresponding to BioProject accession number PRJCA025410 in the Genome Warehouse (GWH) database at National Genomics Data Center (NGCD) website, while those entries are not released until 2026-05-21.

Sorry for asking so many questions. I look forward to hearing from you.

**Comment by yuhaozhangusc** [comment]
Hi, I have the same question about the KP dataset, which cannot be found for Phage genome.
